src/main/java/hudson/plugins/s3/ClientHelper.java

```diff
@@ -120,4 +120,4 @@ private static boolean shouldUseProxy(ProxyConfiguration proxy, String hostname)
 
         return shouldProxy;
     }
-}
+}
```

Comment:
Please keep entpy line at end of file
```suggestion
}

```